Source/Scene/Cesium3DTile.js

```diff
@@ -295,66 +323,62 @@ define([
+            0, 1, 0, 0,
+            0, 0, 1, 0,
+            0, 0, 0, 1
+        );
+        if (boundingVolume.box) {
+            var box = boundingVolume.box;
+            geometry = new RectangleOutlineGeometry({
+                rectangle : new Rectangle(box[0], box[1], box[2], box[3]),
+                height : box[4],
+                extrudedHeight: box[5]
+             });
+        } else if (boundingVolume.sphere) {
+            var sphere = boundingVolume.sphere;
+            geometry = new SphereOutlineGeometry({
+                radius: sphere[3]
+            });
+            Matrix4.fromTranslation(new Cartesian3(sphere[0],sphere[1],sphere[2]), modelMatrix);
+        }
+        return createDebugPrimitive(geometry, color, modelMatrix);
     }
 
 // TODO: remove workaround for https://github.com/AnalyticalGraphicsInc/cesium/issues/2657
-    function workaround2657(rectangle) {
-        return (rectangle[1] !== rectangle[3]) && (rectangle[0] !== rectangle[2]);
+    function workaround2657(boundingVolume) {
+        if (defined(boundingVolume.box)) {
+            var rectangle = boundingVolume.box;
+            return (rectangle[1] !== rectangle[3]) && (rectangle[0] !== rectangle[2]);
+        } else {
+            return true;
+        }
     }
 
     function applyDebugSettings(tile, owner, frameState) {
         // Tiles do not have a content.box if it is the same as the tile's box.
-        var hascontentBox = defined(tile._header.content) && defined(tile._header.content.box);
-
-        var showBox = owner.debugShowBox || (owner.debugShowcontentBox && !hascontentBox);
-        if (showBox && workaround2657(tile._header.box)) {
-            if (!defined(tile._debugBox)) {
-                tile._debugBox = createDebugBox(tile._header.box, hascontentBox ? Color.WHITE : Color.RED);
-            }
-            tile._debugBox.update(frameState);
-        } else if (!showBox && defined(tile._debugBox)) {
-            tile._debugBox = tile._debugBox.destroy();
-        }
-
-        if (owner.debugShowcontentBox && hascontentBox && workaround2657(tile._header.content.box)) {
-            if (!defined(tile._debugcontentBox)) {
-                tile._debugcontentBox = createDebugBox(tile._header.content.box, Color.BLUE);
-            }
-            tile._debugcontentBox.update(frameState);
-        } else if (!owner.debugShowcontentBox && defined(tile._debugcontentBox)) {
-            tile._debugcontentBox = tile._debugcontentBox.destroy();
-        }
+        var hasContentBoundingVolume = defined(tile._header.content) && defined(tile._header.content.boundingVolume);
 
-        if (owner.debugShowBoundingVolume) {
-            if (!defined(tile._debugOrientedBoundingBox)) {
-                tile._debugOrientedBoundingBox = createDebugOrientedBox(tile._orientedBoundingBox, hascontentBox ? Color.WHITE : Color.RED);
+        var showVolume = owner.debugShowBoundingVolume || (owner.debugShowContentBoundingVolume && !hasContentBoundingVolume);
```

Comment:
Slight typo since the name in `Cesium3DTileset` is actually `debugShowContentsBoundingVolume` (extra `s`). It's probably best to edit the variable name in `Cesium3DTileset` and any occurrences in `Cities.html`.